### ISSUE TITLE
API key required to search for location in geopoint enketo widget

### ISSUE THREAD

**Issue by garethbowen** [medic/cht-core#6303]
**Describe the bug**
When trying to search for a location using the geopoint enketo widget it doesn't work and an error is shown in the console.

**To Reproduce**
1. Install the `geopoint.xml` demo form
2. Go to Reports tab
3. Click the New Action button and select the Geopoint form
4. In the search field type a location and click the magnifying glass button
5. Nothing happens. Check the error in the console.

**Expected behavior**
The location is found and selected on the map. No error in the console.

**Logs**
```json
{
   "error_message" : "You must use an API key to authenticate each request to Google Maps Platform APIs. For additional information, please refer to http://g.co/dev/maps-no-account",
   "results" : [],
   "status" : "REQUEST_DENIED"
}
```

**Environment**
- Instance: localhost
- Browser: Chrome
- Client platform: Linux
- App: webapp
- Version: 3.8.0

**Additional context**
The widget fetches the api key from the config.json in enketo-core but we need to be able to configure it per project. Contribute to Enketo as required to allow passing the config or using an unrestricted API (maybe openstreetmaps?). Alternatively we could contribute a change which hides the search box if no api key is given.

**Comment by ChinHairSaintClair** [comment]
Just wondering if there has been any new developments in this area?

**Comment by garethbowen** [comment]
@ChinHairSaintClair I don't think so. I'm really interested in using a free offline first mapping alternative like OSM instead of Google Maps as the API costs and online first approach are always going to be a barrier to use. Is this something you're interested in using?

**Comment by ChinHairSaintClair** [comment]
@garethbowen. Your reasoning is sound, maps are really interesting. I believe we followed a similar approach years back when I worked on an offline-first mobile app in the agriculture field. 
As a result it seems I jumped the gun a bit, our stakeholders confirmed we'll be using an off-app map solution for our reporting.
We won't be needing it at this time.
